### PR TITLE
Rename "batch change" to "DNS change" for portal/portal docs

### PR DESCRIPTION
**Pull request by mitruly**
As we recommend "batch change" as the interface for making DNS changes, renaming "batch change" to "DNS change" is clearer. The API and API docs will keep the existing "batch change" nomenclature.

Note: Portal routes are unchanged.

Changes in this pull request:
- Rename "batch change" in portal
- Rename "batch change" in portal docs

### CODE REVIEW

File: modules/docs/src/main/tut/portal/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The Groups section of the portal is for managing high level user access and abil
 
 [Learn More](groups)
 
-### Batch Changes
-The Batch Changes section of the portal allows users to implement multiple record changes simultaneously and across different zones. To access Batch Changes in the portal select the Batch Changes link in the left column navigation.
+### DNS changes
```
Comment (britneywright) [inline modules/docs/src/main/tut/portal/index.md]: capitalize DNS Changes, it's the proper name that we're using for that part of VinylDNS.

File: modules/docs/src/main/tut/portal/index.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ The Groups section of the portal is for managing high level user access and abil
...
+The DNS changes section of the portal allows users to implement multiple record changes simultaneously and across different zones. To access DNS changes in the portal select the DNS changes link in the left column navigation.
```
Comment (britneywright) [inline modules/docs/src/main/tut/portal/index.md]:
```suggestion
The DNS Changes section of the portal allows users to implement multiple record changes simultaneously and across different zones. To access your DNS changes in the portal select the DNS Changes link in the left column navigation.
```

File: modules/docs/src/main/tut/portal/zones.md [hunk 3]
```diff
@@ -13,13 +13,13 @@ The All Zones tab is intended as a reference, it includes zones you have access
 If needed, you can use the contact information in the list to reach out to the owners of the zones you don't have access to.
 
 **Note** you may have access to specific records in shared zones, but you are not permitted to access those zones in this area.
-Those records are accessible via the [Batch Change](batch-changes) area of the portal.
+Those records are accessible via the [DNS change](dns-changes) area of the portal.
```
Comment (britneywright) [inline modules/docs/src/main/tut/portal/zones.md]:
```suggestion
Those records are accessible via the [DNS Changes](dns-changes) area of the portal.
```

File: modules/portal/public/lib/batch-change/batch-change-new.controller.js [hunk 4]
```diff
@@ -86,7 +86,7 @@
                     var alert = utilityService.success('Successfully Created Batch Change', response, 'createBatchChange: createBatchChange successful');
```
Comment (rebstar6) [inline modules/portal/public/lib/batch-change/batch-change-new.controller.js]: this alert should be `Successfully Created DNS Change` :)

**Comment by rebstar6** [comment]
We may want to rename the files at some point too. Im seeing that's a much larger/riskier change, so keep it out for now, but maybe tech debt that item?

**Comment by mitruly** [comment]
@rebstar6 That's a good point; think I overlooked the `.js` files. I agree - we can table the file rename.

File: modules/portal/public/lib/batch-change/batch-change-new.controller.js (hunk 4 above)
Comment (mitruly) [reply modules/portal/public/lib/batch-change/batch-change-new.controller.js]: Updated this!

File: modules/portal/conf/routes [hunk 5]
```diff
@@ -13,9 +13,9 @@ GET           /groups                            @controllers.FrontendController
 GET           /groups/:gid                       @controllers.FrontendController.viewGroup(gid: String)
 GET           /zones                             @controllers.FrontendController.viewAllZones
 GET           /zones/:zid                        @controllers.FrontendController.viewZone(zid: String)
-GET           /batchchanges                      @controllers.FrontendController.viewAllBatchChanges
-GET           /batchchanges/new                  @controllers.FrontendController.viewNewBatchChange
-GET           /batchchanges/:id                  @controllers.FrontendController.viewBatchChange(id: String)
+GET           /dnschanges                        @controllers.FrontendController.viewAllBatchChanges
```
Comment (britneywright) [inline modules/portal/conf/routes]: can you update the /api/batchchanges endpoints as well

**Pull request merged**